.circleci/config.yml

```diff
@@ -107,6 +107,14 @@ build-steps: &build-steps
           else
             echo 'Skipping tests due to configuration'
           fi
+    - run:
+        name: Verify mksnapshot
+        command: |
+          if [ "$RUN_TESTS" == "true" ] && [ "$ELECTRON_RELEASE" == "1" ]; then
+            echo 'Verifying mksnapshot on release build'
+          else
+            echo 'Skipping mksnapshot tests due to configuration'
+          fi
```

Comment:
It looks like this just prints some stuff - was this intended or no?

Reply:
Doh! 🤦‍♂️